lib/ansible/modules/cloud/amazon/ec2_vpc_peer.py

```diff
@@ -276,6 +303,7 @@ def remove_peer_connection(client, module):
     if not pcx_id:
         params['VpcId'] = module.params.get('vpc_id')
         params['PeerVpcId'] = module.params.get('peer_vpc_id')
+        params['PeerRegion'] = module.params.get('peer_region')
```

Comment:
I've noticed a bug [here](https://github.com/ansible/ansible/pull/34036/files#diff-e6bd4ac11744cf53baafe01a2f8c0ed2L288). `params` is being populated and then is reused to delete the peering connection and can lead to errors like: `Unknown parameter in input: \"PeerRegion\", must be one of: DryRun, VpcPeeringConnectionId\nUnknown parameter in input: \"PeerVpcId\", must be one of: DryRun, VpcPeeringConnectionId\n`
delete_vpc_peering_connection() should only take VpcPeeringConnectionId=pcx_id - do you mind fixing that?